### ISSUE TITLE
Table cells have ugly default focus outline when given mouse focus

### ISSUE THREAD

**Issue by lazd** [adobe/spectrum-css#453]
## Description

/title

## Steps to reproduce

1. Go to http://spectrum-css-archives.s3-website.us-east-2.amazonaws.com/2.15.1/docs/table.html
2. Press tab lots
3. Observe ugly outline

## Expected behavior

It should have the pretty focus outline, if at all. Need design input.

![image](https://user-images.githubusercontent.com/201344/70838413-e8804900-1dbc-11ea-8f91-6af397e02b5e.png)

## Screenshots

![image](https://user-images.githubusercontent.com/201344/70838398-dd2d1d80-1dbc-11ea-97a4-49e7c60664e5.png)

## Environment
 - **Spectrum CSS version:** 2.15.1
 - **Browser(s) and OS(s):** Chrome on macOS

**Comment by GarthDB** [comment]
Looks like it's fixed.
![image](https://user-images.githubusercontent.com/125516/93129106-b8d4d900-f68d-11ea-8a8c-dda2514f8f12.png)
